### PR TITLE
ECE120 course page completed

### PR DESCRIPTION
**Pull request by upai02**
Fixed with changes mentioned in last PR

### CODE REVIEW

File: docs/Course Wiki/ECE Course Offerings/ECE120.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# ECE120
+
```
Comment (alex-m-zhang) [inline docs/Course Wiki/ECE Course Offerings/ECE120.md:2]: Add a thing here like this:

ECE120 (course title) is a 3-credit-hour course that is specifically required for both EEs and CEs are part of the Electrical and Computer Engineering Core. It is offered in the fall, spring, and summer.

File: docs/Course Wiki/ECE Course Offerings/ECE120.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Do not be afraid to ask questions in lecture and discussion! ECE120 covers a vast amount of topics to provide a broad overview of computing, so understanding the material will be helpful for every future computing-related ECE course in some way or another. Make it a goal to ask a question every lecture (this is easily doable if you pay attention, except maybe the first lecture).
+
+## Life After
+
+After completing ECE120, ECE students are encouraged to take ECE220, the second core computer engineering course, as soon as possible. ECE220 builds onto concepts introduced at the tail end of ECE120, which in turn build on concepts covered throughout the rest of the course; therefore, having a strong understanding of ECE120 material is important prior to taking ECE220. In due time after taking the core ECE courses, ECE385 offers a continuation of logical design elements, as well as hardware and software implementations of applicable scenarios. ECE411 offers an even more in-depth continuation of the computer organization concepts covered in ECE120 and ECE385, and is generally taken by students in their senior year after completing the rest of the computer engineering core classes.
```
Comment (alex-m-zhang) [inline docs/Course Wiki/ECE Course Offerings/ECE120.md]: Link 220 here on first mention. Link 385 here on first mention. Link 411 as well.

**Review by alex-m-zhang — CHANGES_REQUESTED** [review]
Last few things - good work!

File: docs/Course Wiki/ECE Course Offerings/ECE120.md (hunk 1 above)
Comment (upai02) [reply docs/Course Wiki/ECE Course Offerings/ECE120.md:2]: Added on latest commit

File: docs/Course Wiki/ECE Course Offerings/ECE120.md (hunk 2 above)
Comment (upai02) [reply docs/Course Wiki/ECE Course Offerings/ECE120.md]: Added on latest commit

**Review by alex-m-zhang — APPROVED** [review]
LGTM! @lnguy7 you may merge on your approval.

**Review by lnguy7 — APPROVED** [review]
Forogr to approve

**Pull request merged**